meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -29,6 +37,8 @@ runSupervisor() {
         -e "LED_FILE=${LED_FILE}" \
         -e "LISTEN_PORT=$LISTEN_PORT" \
         -e "SUPERVISOR_IMAGE=${SUPERVISOR_IMAGE}:${SUPERVISOR_TAG}" \
+        -e "BALENA_ROOT_CA=$BALENA_DECODED_ROOT_CA" \
+        -e "NODE_EXTRA_CA_CERTS=$(if [ ! -z "${BALENA_ROOT_CA}" ]; then echo "/etc/ssl/certs/balenaRootCA.pem"; fi)" \
```

Comment:
I would simplify this as well by defining `NODE_EXTRA_CA_CERTS` in the `if [ ! -z "${BALENA_ROOT_CA}" ]; then` above.